concourse/scripts/pipecleaner.py

```diff
@@ -137,6 +164,18 @@ def check_pipeline(self, data):
                         for i in item['config']['outputs']:
                             output_resources.add(i['name'])
 
+                    if 'run' in item['config']:
+                        if item['config']['run']['path'] in ['sh', 'bash', 'dash', 'ksh']:
```

Comment:
Ah right.  Yes makes sense, will think of how to do that, I think it's another class of check.